tsconfig.json

```diff
@@ -9,6 +9,7 @@
   "compilerOptions": {
     "outDir": "dist-types",
     "rootDir": ".",
-    "useUnknownInCatchVariables": false
+    "useUnknownInCatchVariables": false,
+    "jsx": "react"
```

Comment:
This isn't actually needed right?

Reply:
Yeah, i'll remove it. I think it was needed it some of old tests, which i removed with old code.